doc/modules/model_evaluation.rst

```diff
@@ -85,6 +85,7 @@ Scoring                           Function
 **Regression**
 'explained_variance'              :func:`metrics.explained_variance_score`
 'neg_mean_absolute_error'         :func:`metrics.mean_absolute_error`
+'neg_mape'                        :func:`metrics.mean_absolute_percentage_error`
```

Comment:
Why not spell it out fully here since like all the other metrics? i.e. neg_mean_absolute_percentage_error

Reply:
I clarified in the PR description above that the name has to be chosen/voted by all of us. Initially I used `neg_mean_absolute_percentage_error` but then, since `mape` is already a famous acronym which, also, makes the metric cleaner, I chose to switch to neg_mape. However, we can change to the otherwise, If most of us think that the right thing to do.

Reply:
I would be in favour of `neg_mean_absolute_error` version personally. It is more consistent with `neg_mean_absolute_error` and more consistent with the metric name ( `metrics.mean_absolute_percentage_error`). Happy to hear what others think.

Reply:
I would also be in favor using the explicit expanded name by default and introduce `neg_mape` as an alias as we do for `neg_mse`.

Reply:
Actually we do not have `neg_mse`. I thought we had.